assets/templates/index.html

```diff
@@ -0,0 +1,61 @@
+{{define "metrics"}}
+<!DOCTYPE html>
```

Comment:
[CHANGE REQUEST] Please change the layout
https://github.com/golang-standards/project-layout/tree/master/web

Reply:
done